src/System.ServiceProcess.ServiceController/tests/TestServiceProvider.cs

```diff
@@ -63,12 +64,13 @@ public TestServiceProvider()
             CreateTestServices();
         }
 
-        public TestServiceProvider(string serviceName)
+        public TestServiceProvider(string serviceName, string userName = "NT AUTHORITY\\LocalService")
```

Comment:
nit: I'm assuming this is already in a const string somewhere else in the test assembly;  assuming so, can this be changed to be string userName = TestServiceProvider.LocalServiceName instead